doc/contributing/docs/markup/intro.rst

```diff
@@ -11,11 +11,11 @@ Text can be organized in bullet-lists:
 
 ..  code-block:: rst
 
-    - This is a bullet list.
+    *   This is a bullet list.
 
-    - Bullets can be "*", "+", or "-".
+    *   Bullets can be "*", "+", or "-".
 
-        * Lists can be nested. And it is good to indent them with 4 spaces.
+        -   Lists can be nested. And it is good to indent them with 4 spaces.
```

Comment:
As for 4 space indent, I would suggest changing the wording and making it "a must" instead of "nice to have" as it says now. Because we have this in our doc guidelines https://www.tarantool.io/en/doc/latest/contributing/docs/markup/intro/#indentation

Reply:
I don't think it is necessary here)